src/fu-util.c

```diff
@@ -1848,8 +1853,14 @@ fu_util_update_all (FuUtilPrivate *priv, GError **error)
 		g_autoptr(GError) error_local = NULL;
 
 		/* not going to have results, so save a D-Bus round-trip */
-		if (!fwupd_device_has_flag (dev, FWUPD_DEVICE_FLAG_SUPPORTED))
+		if (!fwupd_device_has_flag (dev, FWUPD_DEVICE_FLAG_UPDATABLE))
+			continue;
+		if (!fwupd_device_has_flag (dev, FWUPD_DEVICE_FLAG_SUPPORTED)) {
+			/* TRANSLATORS: message letting the user know no device upgrade available due to missing on LVFS
+			* %1 is the device name */
+			g_printerr (_("• %s has no available firmware updates\n"), fwupd_device_get_name (dev));
```

Comment:
I don't think the translators are happy when you include `\n` in the `_()` string. You might need to do a `g_autofree gchar *tmp = g_strdup_printf` thing...

Reply:
fixed